docs/en/Modules/Docs.md

```diff
@@ -176,51 +159,57 @@ An ABP module must declare `[DependsOn]` attribute if it has a dependency upon a
       }
   ```
 
-
-
 ### 4- Database Integration
 
 #### 4.1- Entity Framework Integration
 
-If you choose Entity Framework as your database provider, you need to configure the Docs Module in your DbContext. To do this;
+If you choose Entity Framework as your database provider, you need to configure the Docs Module. To do this;
 
-- Open `MyProjectDbContext.cs` and add `modelBuilder.ConfigureDocs()` to the `OnModelCreating()` 
+- Open `QaDocDbContextModelCreatingExtensions.cs` and add `modelBuilder.ConfigureDocs()` to the `ConfigureMyProject()`.
 
   ```csharp
-  [ConnectionStringName("Default")]
-  public class MyProjectDbContext : AbpDbContext<MyProjectDbContext>
+  public static class QaDocDbContextModelCreatingExtensions
   {
-      public MyProjectDbContext(DbContextOptions<MyProjectDbContext> options)
-          : base(options)
+      public static void ConfigureMyProject(this ModelBuilder builder)
       {
-  
+          Check.NotNull(builder, nameof(builder));
+
+          /* Configure your own tables/entities inside here */
+
+          //builder.Entity<YourEntity>(b =>
+          //{
+          //    b.ToTable(QaDocConsts.DbTablePrefix + "YourEntities", QaDocConsts.DbSchema);
+
+          //    //...
+          //});
+
+          builder.ConfigureDocs();
       }
```

Comment:
I think the `ConfigureDocs `method should be located in the `XXXMigrationsDbContext `class. Right?